app/src/main/java/de/westnordost/streetcomplete/quests/building_type/BuildingType.kt

```diff
@@ -0,0 +1,76 @@
+package de.westnordost.streetcomplete.quests.building_type
+
+enum class BuildingType(val osmTag: String?, val subTypes: List<BuildingType>? = null) {
```

Comment:
Interesting idea, to include both key and value information in here. I think this makes sense. However, then it would make more sense to have two fields for that: `osmKey` and `osmValue`